cmake/flash/CMakeLists.txt

```diff
@@ -77,10 +77,15 @@ function(create_runners_yaml)
     endif()
   endforeach()
 
+  get_property(runners_yaml_extra GLOBAL PROPERTY ZEPHYR_MODULE_RUNNERS_YAML)
+
   # Write the final contents and set its location in the cache.
   file(GENERATE OUTPUT "${runners_yaml}" CONTENT
     $<TARGET_PROPERTY:runner_yml_props_target,yaml_contents>)
-  set(ZEPHYR_RUNNERS_YAML "${runners_yaml}" CACHE INTERNAL
+  get_property(EXTRA_ZEPHYR_RUNNERS_YAML
```

Comment:
The reason for `EXTRA_ZEPHYR_RUNNERS_YAML` to be upper case, is because existing `ZEPHYR_RUNNERS_YAML` is upper case, and those two local variables are belonging together.
The reason for `ZEPHYR_RUNNERS_YML` property on `zephyr_property_target` to be uppercase, is because properties on targets are usually upper case:
https://cmake.org/cmake/help/latest/manual/cmake-properties.7.html#properties-on-targets